packages/integrations/vercel/package.json

```diff
@@ -50,7 +50,7 @@
   "dependencies": {
     "@astrojs/internal-helpers": "workspace:*",
     "@vercel/analytics": "^1.5.0",
-    "@vercel/edge": "^1.2.1",
+    "@vercel/functions": "^2.2.2",
```

Comment:
The package changed here. Is that intentional?

Reply:
Yes, the edge package was deprecated. Forgot to add it to the description